### ISSUE TITLE
Editorconfig-checker is not listed in the Linters array

### ISSUE THREAD

**Issue by ferrarimarco** [github/super-linter#543]
**Describe the bug**
`editorconfig-checker` [is not listed in the `LINTER_ARRAY` array](https://github.com/github/super-linter/blob/master/lib/linter.sh#L127).

**To Reproduce**

1. Enable debug output
1. Run super-linter
1. Inspect the `Linter version info` section in the output

**Expected behavior**
`editorconfig-checker` should be listed in the `LINTER_ARRAY` array.

**Screenshots**
`editorconfig-checker` is not reported in the `Linter version info` output. This is what I have:

```
Step #0 - "super-linter": 2020-08-10 13:24:30 [DEBUG ]   ---------------------------------------------
Step #0 - "super-linter": 2020-08-10 13:24:30 [DEBUG ]   Linter Version Info:
Step #0 - "super-linter": 2020-08-10 13:24:32 [DEBUG ]   Successfully found version for [ansible-lint]: ansible-lint 4.2.0
Step #0 - "super-linter": 2020-08-10 13:24:32 [DEBUG ]   Successfully found version for [arm-ttk]: ModuleVersion = 0.3
Step #0 - "super-linter": 2020-08-10 13:24:32 [DEBUG ]   Successfully found version for [asl-validator]: 1.7.0
Step #0 - "super-linter": 2020-08-10 13:24:34 [DEBUG ]   Successfully found version for [cfn-lint]: cfn-lint 0.34.1
Step #0 - "super-linter": 2020-08-10 13:24:34 [DEBUG ]   Successfully found version for [clj-kondo]: clj-kondo v2020.07.29
Step #0 - "super-linter": 2020-08-10 13:24:34 [DEBUG ]   Successfully found version for [coffeelint]: 3.2.10
Step #0 - "super-linter": 2020-08-10 13:24:34 [DEBUG ]   Successfully found version for [dart]: Dart VM version: 2.8.4 (stable) (Wed Jun 3 12:26:04 2020 +0200) on "linux_x64"
Step #0 - "super-linter": 2020-08-10 13:24:35 [DEBUG ]   Successfully found version for [dockerfilelint]: 1.5.0
Step #0 - "super-linter": 2020-08-10 13:24:35 [DEBUG ]   Successfully found version for [dotenv-linter]: dotenv-linter 2.1.0
Step #0 - "super-linter": 2020-08-10 13:24:35 [DEBUG ]   Successfully found version for [eslint]: v7.6.0
Step #0 - "super-linter": 2020-08-10 13:24:35 [DEBUG ]   Successfully found version for [flake8]: 3.8.3 (mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.8.5 on Linux
Step #0 - "super-linter": 2020-08-10 13:24:36 [DEBUG ]   Successfully found version for [golangci-lint]: golangci-lint has version 1.30.0 built from 45b90f6 on 2020-08-03T03:10:34Z
Step #0 - "super-linter": 2020-08-10 13:24:36 [DEBUG ]   Successfully found version for [hadolint]: Haskell Dockerfile Linter v1.18.0-6-ga0d655d
Step #0 - "super-linter": 2020-08-10 13:24:36 [DEBUG ]   Successfully found version for [htmlhint]: 0.14.1
Step #0 - "super-linter": 2020-08-10 13:24:37 [DEBUG ]   Successfully found version for [jsonlint]: 1.6.3
Step #0 - "super-linter": 2020-08-10 13:24:38 [DEBUG ]   Successfully found version for [ktlint]: 0.37.2
Step #0 - "super-linter": 2020-08-10 13:24:38 [DEBUG ]   Successfully found version for [lua]: lua: unrecognized option '--version' usage: lua [options] [script [args]] Available options are:   -e stat  execute string 'stat'   -i       enter interactive mode after executing 'script'   -l name  require library 'name' into global 'name'   -v       show version information   -E       ignore environment variables   --       stop handling options   -        stop handling options and execute stdin
Step #0 - "super-linter": 2020-08-10 13:24:38 [DEBUG ]   Successfully found version for [markdownlint]: 0.23.2
Step #0 - "super-linter": 2020-08-10 13:24:41 [DEBUG ]   Successfully found version for [npm-groovy-lint]: GroovyLint: Started CodeNarc Server GroovyLint: Successfully processed CodeNarc:  CodeNarc version 1.6.1  npm-groovy-lint version 6.1.1  Embeds: CodeNarc version 1.6.1 - Groovy version 3.0.5 (superlite)
Step #0 - "super-linter": 2020-08-10 13:24:41 [DEBUG ]   Successfully found version for [perl]:  This is perl 5, version 30, subversion 3 (v5.30.3) built for x86_64-linux-thread-multi  Copyright 1987-2020, Larry Wall  Perl may be copied only under the terms of either the Artistic License or the GNU General Public License, which may be found in the Perl 5 source kit.  Complete documentation for Perl, including FAQ lists, should be found on this system using "man perl" or "perldoc perl".  If you have access to the Internet, point your browser at http://www.perl.org/, the Perl Home Page. 
Step #0 - "super-linter": 2020-08-10 13:24:41 [DEBUG ]   Successfully found version for [protolint]: --version not supported
Step #0 - "super-linter": 2020-08-10 13:24:41 [DEBUG ]   Successfully found version for [pwsh]: PowerShell 7.0.3
Step #0 - "super-linter": 2020-08-10 13:24:42 [DEBUG ]   Successfully found version for [pylint]: pylint 2.5.3 astroid 2.4.2 Python 3.8.5 (default, Aug  4 2020, 04:11:56)  [GCC 9.3.0]
Step #0 - "super-linter": 2020-08-10 13:24:42 [DEBUG ]   Successfully found version for [raku]: This is Rakudo version 2020.07 built on MoarVM version 2020.07 implementing Raku 6.d.
Step #0 - "super-linter": 2020-08-10 13:24:43 [DEBUG ]   Successfully found version for [rubocop]: 0.82.0
Step #0 - "super-linter": 2020-08-10 13:24:43 [DEBUG ]   Successfully found version for [shellcheck]: ShellCheck - shell script analysis tool version: 0.7.1 license: GNU General Public License, version 3 website: https://www.shellcheck.net
Step #0 - "super-linter": 2020-08-10 13:24:44 [DEBUG ]   Successfully found version for [spectral]: 5.4.0
Step #0 - "super-linter": 2020-08-10 13:24:44 [DEBUG ]   Successfully found version for [standard]: 14.3.4
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   Successfully found version for [stylelint]: 13.6.1
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   Successfully found version for [terrascan]: Terrascan v0.2.3
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   Successfully found version for [tflint]: TFLint version 0.18.0
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   Successfully found version for [xmllint]: xmllint: using libxml version 20910    compiled with: Threads Tree Output Push Reader Patterns Writer SAXv1 FTP HTTP DTDValid HTML Legacy C14N Catalog XPath XPointer XInclude Iconv ISO8859X Unicode Regexps Automata Schemas Schematron Modules Debug Zlib Lzma 
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   Successfully found version for [yamllint]: yamllint 1.24.2
Step #0 - "super-linter": 2020-08-10 13:24:45 [DEBUG ]   ---------------------------------------------
```

**Additional context**
Discovered while debugging #491

**Comment by nemchik** [comment]
Good catch.

**Comment by ferrarimarco** [comment]
@admiralAwkbar I see this in my logs (after updating to `3.6.0`):

```
[editorconfig-checker]: Failed to get version info for:
```

Guess there's something wrong in how editorconfig-checker is queried for its own version?

**Comment by admiralAwkbar** [comment]
fixing now
